test/scripts/openshift-ci/run-e2e-tests.sh

```diff
@@ -71,19 +71,24 @@ if [ "$1" != "raw" ]; then
 fi
 
 echo "Installing KServe with Minio"
-kustomize build $PROJECT_ROOT/config/overlays/test | \
-  sed "s|kserve/storage-initializer:latest|${STORAGE_INITIALIZER_IMAGE}|" | \
-  sed "s|kserve/agent:latest|${KSERVE_AGENT_IMAGE}|" | \
-  sed "s|kserve/router:latest|${KSERVE_ROUTER_IMAGE}|" | \
-  sed "s|kserve/kserve-controller:latest|${KSERVE_CONTROLLER_IMAGE}|" | \
-  oc apply -f -
-
-# Patch the inferenceservice-config ConfigMap, when running RawDeployment tests
- if [ "$1" == "raw" ]; then
+kustomize build $PROJECT_ROOT/config/overlays/test |
+  sed "s|kserve/storage-initializer:latest|${STORAGE_INITIALIZER_IMAGE}|" |
+  sed "s|kserve/agent:latest|${KSERVE_AGENT_IMAGE}|" |
+  sed "s|kserve/router:latest|${KSERVE_ROUTER_IMAGE}|" |
+  sed "s|kserve/kserve-controller:latest|${KSERVE_CONTROLLER_IMAGE}|" |
+  oc apply --server-side=true -f -
+
+Patch the inferenceservice-config ConfigMap, when running RawDeployment tests
+if [ "$1" == "raw" ]; then
   export OPENSHIFT_INGRESS_DOMAIN=$(oc get ingresses.config cluster -o jsonpath='{.spec.domain}')
   oc patch configmap inferenceservice-config -n kserve --patch-file <(cat config/overlays/test/configmap/inferenceservice-openshift-ci-raw.yaml | envsubst)
   oc delete pod -n kserve -l control-plane=kserve-controller-manager
- fi
+
+  oc patch DataScienceCluster test-dsc --type='json' -p='[{"op": "replace", "path": "/spec/components/kserve/defaultDeploymentMode", "value": "RawDeployment"}]'
+else
+  export OPENSHIFT_INGRESS_DOMAIN=$(oc get ingresses.config cluster -o jsonpath='{.spec.domain}')
+  oc patch configmap inferenceservice-config -n kserve --patch-file <(cat config/overlays/test/configmap/inferenceservice-openshift-ci-serverless.yaml | envsubst)
```

Comment:
I'm confused about why you need to configure the ingress domain here.

In the [comment docs](https://github.com/kserve/kserve/blob/b4ce5d6ec506bd064fda87b230b25b23f21397c5/config/configmap/inferenceservice.yaml#L230-L233), it is stated that this is only relevant for `RawDeployment`, but Raw is covered in the other branch of the `if`